libs/execution_context/ebpf_maps.c

```diff
@@ -496,20 +496,12 @@ _check_value_type(_In_ const ebpf_core_map_t* outer_map, _In_ const ebpf_core_ob
     return allowed;
 }
 
-// Validate that a value handle is appropriate for this map,
-// and if so, return a pointer to the object with that handle.
-static ebpf_result_t
-_get_map_value_object(
-    _In_ const ebpf_core_map_t* map,
-    ebpf_handle_t value_handle,
-    ebpf_object_type_t value_type,
-    _Outptr_ ebpf_core_object_t** value_object_result)
+// Validate that a value object is appropriate for this map.
+static _Requires_lock_held_(object_map->lock) ebpf_result_t _validate_map_value_object(
+    _In_ const ebpf_core_object_map_t* object_map, ebpf_object_type_t value_type, _In_ ebpf_core_object_t* value_object)
```

Comment:
`value_object` should be `const` as well.

Reply:
Fixed. Also removed const from "object_map" as it is being modified in this function.